apps/rush-lib/src/cli/RushCommandLineParser.ts

```diff
@@ -81,7 +82,15 @@ export class RushCommandLineParser extends CommandLineParser {
     try {
       const rushJsonFilename: string | undefined = RushConfiguration.tryFindRushJsonLocation({
         startingFolder: this._rushOptions.cwd,
-        showVerbose: !Utilities.shouldRestrictConsoleOutput()
+        showVerbose: !Utilities.shouldRestrictConsoleOutput(),
+        logger: (message: string) => {
+          const existing: string | undefined = this._rushOptions.startupBanner;
+          if (existing) {
+            this._rushOptions.startupBanner = `${existing}${os.EOL}${message}`;
```

Comment:
I assume this is to match the behavior of `console.log`?

Reply:
Was, yes, but now skipping this log altogether.